pkg/metrics/metrics_controller.go

```diff
@@ -101,16 +101,21 @@ var (
 		Name:      "controller_bgp_internal_peers_sync_time",
 		Help:      "Time it took to sync internal bgp peers",
 	}, []string{})
-	ControllerBGPadvertisementsReceived = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	ControllerBGPadvertisements = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: namespace,
-		Name:      "controller_bgp_advertisements_received",
-		Help:      "Time it took to sync internal bgp peers",
-	}, []string{})
+		Name:      "controller_bgp_advertisements",
+		Help:      "Number of BGP advertisements received and sent",
```

Comment:
small nit: can we make this `received and advertised` to be consistent with code?

Reply:
ofc